modules/classroom/manifests/params.pp

```diff
@@ -77,5 +77,23 @@
 
   if versioncmp($::pe_version, '3.3.0') < 0 {
     fail("Your Puppet Enterprise installation is out of date. ${download}")
+
+  #Windows active directory setup parameters
```

Comment:
I'd like this block to start just below line 50, and it should be in the format `$var = 'value'`.